lib/device/library/linux.rb

```diff
@@ -17,5 +17,56 @@ def get_device_root_dir(volume_name)
       end
       nil
     end
+
+    # 端末のデバイスファイル名と uhelper オプションの値を取得
+    def device_mount_info(volume_name)
+      device_root = get_device_root_dir(volume_name)
+      raise Device::CantEject, "端末が接続されていません" if device_root.nil?
+
+      pattern = %r!^(/dev/[^ ]+) .* #{device_root} .*\Wuhelper=(\w+)!
+      open("|mount") do |f|
+        while line = f.gets
+          if line =~ pattern
+            return [$1, $2]
+          end
+        end
+      end
+      [nil, nil]
+    end
+
+    def ejectable?(volume_name)
+      begin
+        case device_mount_info(volume_name)[1]
+        when "udisks", "udisks2"
+          true
+        else
+          false
+        end
+      rescue
```

Comment:
__[rubocop]__
* Style/RescueEnsureAlignment: `rescue` at 45, 6 is not aligned with `end` at 48, 4.

<!-- Comment by [SideCI](https://sideci.com) -->